azure_functions_worker/utils/dependency.py

```diff
@@ -200,6 +206,12 @@ def reload_all_namespaces_from_customer_deps(cls, cx_working_dir: str):
         cls._remove_from_sys_path(cls.worker_deps_path)
         cls._add_to_sys_path(cls.cx_deps_path, True)
         cls._add_to_sys_path(working_directory, False)
+
+        # Deprioritize worker dependencies but don't completely remove it
+        # Otherwise, it will break some really old function apps, those
+        # don't have azure-functions module in .python_packages
+        cls._add_to_sys_path(cls.worker_deps_path, False)
```

Comment:
If possible - add the PR number in both comments (one above as well). That'll redirect people to the PR/issue which would have more verbose details of the issue.